Resources/Textures/Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi/meta.json

```
@@ -31,6 +31,9 @@
         {
             "name": "base-6"
         },
+        {
+            "name": "practice" //ss-220 sprite fix
```

```suggestion
            "name": "practice"
```
в мете не нужны комменты, никто этого никогда не делает, да и выглядит странно, так что не стоит